tests/PySys/software_management_end_to_end/sm_mapper_fail_and_reconnect/run.py

```diff
@@ -6,25 +6,25 @@
 from environment_tedge import TedgeEnvironment
 
 """
-Validate the tedge-mapper-sm-c8y does not loose last message from tedge-agent when it fails and comes back
+Validate the tedge-mapper-c8y does not loose last message from tedge-agent when it fails and comes back
 
 Given a configured system
 When `rolldice` package is installed
 when a subscriber is started as `sudo tedge mqtt sub 'c8y/s/us'`
 When tedge_agent is started as `sudo systemctl start tedge-agent.service`
-When sm mapper is started as `sudo systemctl start tedge-mapper-sm-c8y.service`
+When c8y mapper is started as `sudo systemctl start tedge-mapper-c8y.service`
 When send a delete operation `sudo tedge mqtt pub "c8y/s/ds" "528,tedge,rolldice,,,delete"`
-When sm mapper is stopped `sudo systemctl stop tedge-mapper-sm-c8y.service`
+When c8y mapper is stopped `sudo systemctl stop tedge-mapper-c8y.service`
 Wait for sometime for operation to be completed and agent to push the operation result.
-When sm mapper is restarted `sudo systemctl restart tedge-mapper-sm-c8y.service`
-Now sm mapper receives the last update result message, process and forwards it to the cloud on `c8y/s/us`
+When c8y mapper is restarted `sudo systemctl restart tedge-mapper-c8y.service`
+Now c8y mapper receives the last update result message, process and forwards it to the cloud on `c8y/s/us`
 Then validate subscriber output for `501,c8y_SoftwareUpdate`, for the status of operation
 Then validate subscriber output for `503,c8y_SoftwareUpdate` for final result of operation
 Then test has passed
 """
 
 
-class SmMapperC8yReceiveLastMessageOnRestart(BaseTest):
+class MapperC8yReceiveLastMessageOnRestart(BaseTest):
```

Comment:
Can you also rename `pysystest.xml` of this test? It is the last one having `sm-c8y` ;)

This part.
```
	<class name="SmMapperC8yReceiveLastMessageOnRestart" module="run"/>
```